src/execution/execute.d.ts

```diff
@@ -89,7 +127,9 @@ export interface ExecutionArgs {
  *
  * Accepts either an object with named arguments, or individual arguments.
  */
-export function execute(args: ExecutionArgs): PromiseOrValue<ExecutionResult>;
+export function execute(
+  args: ExecutionArgs,
+): PromiseOrValue<ExecutionResult | AsyncIterable<AsyncExecutionResult>>;
```

Comment:
AsyncIterableIterator might be the better return type as it can easily be looped over with a for await of loop

Reply:
Updated to `AsyncIterableIterator`